docs/userguide/networking/default_network/container-basics.md

```diff
@@ -0,0 +1,106 @@
+<!--[metadata]>
++++
+draft=true
+title = "Configure container DNS"
+description = "Learn how to configure DNS in Docker"
+keywords = ["docker, bridge, docker0, network"]
+[menu.main]
+parent = "smn_networking_def"
++++
+<![end-metadata]-->
+
+# How the default network
+
+The information in this section explains configuring container DNS within tthe Docker default bridge. This is a `bridge` network named `bridge` created
+automatically when you install Docker.
```

Comment:
Gang, this material is a DRAFT (see metadata) it doesn't get built into docs.docker.  I have it here in case I need to pull from it or reinstate it at a later date.  I'll add another comment.